### ISSUE TITLE
[Adaptive Cards -Activity Update with Labels]: Ensures elements with ARIA roles have all required ARIA attributes

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8682]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**
URL: https://adaptivecards.io/samples/ActivityUpdateWithLabels.html
OS Version: 23H2 (OS Build 25931.1000)
Browser Version: Edge Dev 118.0.2060.1
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on the edge dev browser.
2. Run the tool and observe the issue.

**Actual Result:**
'Publish Adaptive Card Schema' label does not have required ARIA attribute.

**Expected Result:**
'Publish Adaptive Card Schema' label should have required ARIA attribute.

**Element path:**
 .ac-textBlock[role="heading"]

**Snippet:**
 <div class="ac-textBlock" role="heading" style="overflow: hidden; font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, Helvetica, sans-serif; font-size: 17px; color: rgb(0, 0, 0); font-weight: 600; text-align: start; line-height: 22.61px; white-space: nowrap; text-overflow: ellipsis; box-sizing: border-box; flex: 0 0 auto;">

**How to fix:** 
Fix any of the following:
  Required ARIA attribute not present: aria-level

**Note:**
Same issue is repro under below URL's as well.
1. https://adaptivecards.io/samples/InputFormWithLabels.html
2. https://adaptivecards.io/samples/CalendarReminderWithLabels.html
3. https://adaptivecards.io/samples/ExpenseReportWithLabels.html
4. https://adaptivecards.io/samples/SimpleFallback.html

**User Impact:**
This will impact the screen reader user if the proper ARIA attributes are not defined for the label as user will not be able to know about the actual role or use of the labels.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![WCAG4 1 2- Ensures elements with ARIA roles have required ARIA attribute](https://github.com/microsoft/AdaptiveCards/assets/95913748/3ccac71a-a6c5-44bf-aeaa-3a70f73fe1e0)

![WCAG4 1 2- Ensures elements with ARIA roles have required ARIA attributes](https://github.com/microsoft/AdaptiveCards/assets/95913748/098647a4-0a85-4fdb-bbad-3a1c26628843)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
As the above-mentioned URL is not available and the on the same UI, this issue is not repo, hence closing this issue.

 
![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/6df0c812-ecfa-4f44-9277-09b08acef47c)
